superdesk/default_settings.py

```diff
@@ -26,7 +26,7 @@
 try:
     from urllib.parse import urlparse
 except ImportError:
-    from urlparse import urlparse
+    from urlparse import urlparse # type: ignore
```

Comment:
As we have moved to Python 3.6, can we just use `urllib.parse`?

Reply:
makes sense